1906-03-05.xml

```diff
@@ -1221,13 +1220,5796 @@
+                        Placards issued by the Mahdi who overthrew Egyptian rule in the Eastern Sudan were found posted in a street of Bida, 
+                        two thousand miles west of Khartoum; and when Lord Kitchener began his reconquest of the Sudan, the Khalifa urged 
+                        the Sultan of Sokoto to make a diversion in his favour by driving the Christians out of Nigeria. The pressure on 
+                        this fanatical Mahomedan population in the Sudan has now for many years been kept up from several sides at once 
+                        from the Egyptian Sudan on the east, from the Mediterranean downwards by France, and from the Gulf of Guinea upwards 
+                        through Nigeria by England. Almost all the Mahomedan tribes have a common grievance, and if a counterstroke were 
+                        attempted the only wonder would be that it had been delayed so long. The present movement may not turn out to be so 
+                        serious, but, whether it is or not, a general rising of the Mahomedan tribes in the Sudan must always be accounted one 
+                        of the permanent dangers to the Anglo-French settlement in North Africa. It would not be in the least fanciful to suspect 
+                        some connection between the rising in Sokoto and the Conference of Algeciras.</p>
+                </div>
+                <div type="item">
+                    <head>BRITISH RULE IN CYPRUS.</head>
+                    <p>The "Courrier European" publishes an article by M. S. Skiadaresis, giving an account of the condition of affairs in Cyprus. 
+                        The writer asserts that British rule has not improved the economic condition of the island, but has, of anything, made it worse. 
+                        He ascribes this failure, first, to the saddling of the island with the payment of the 93,000 per annum which England agreed to 
+                        make over to Turkey by the Treaty of June 4th, 1878; second, to the costly administration by high-salaried British officials; 
+                        and, third, to the wasteful and sometimes (as in the case of the recent railway construction totally mistaken way of carrying out 
+                        public works. The writer acknowledges that now, on account of this failure on the part of the British to improve matters, 
+                        there in the island a widespread national feeling in favour of anuexation to Greece, and he expresses the hope that the new High Commissioner, 
+                        Sir Charles King-Harman, to whom he gives every credit for his generosity and consideration in treating the natives, may yet plead the 
+                        cause of the Cypriots before his Government.</p>
+                </div>
+            </div>
+            <div type="page" n="3">
+                <div type="item">
```

Comment:
`<div type="section" element="local">`, then make every item within the section its own item with headline.